R/main.R

```diff
@@ -8,19 +8,26 @@ main_args <- function(args) {
   orderly.server [options] <path>
 
 Options:
-  --port=PORT       Port to run on [default: 8321]
-  --host=HOST       IP address owned by this server [default: 0.0.0.0]
-  --no-ref          Prevent git reference switching
-  --go-signal=PATH  Relative path for go signal
-  --workers=WORKERS Number of workers to spawn [default: 0]"
+  --port=PORT                    Port to run on [default: 8321]
+  --host=HOST                    IP address owned by this server [default: 0.0.0.0]
+  --no-ref                       Prevent git reference switching
+  --go-signal=PATH               Relative path for go signal
+  --workers=WORKERS              Number of workers to spawn [default: 0]
+  --backup-period=BACKUP_PERIOD  How frequently should backup be run negative for no backup [default: 600]"
```

Comment:
What does a period of 0 do?

Reply:
If it is 0 it will backup on every API request, could change it to 0 to mean never backup instead.

Reply:
I think never backup is more sensible